pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -346,10 +346,11 @@ func ChooseRandomRegion(regions ...CloudRegion) (CloudRegion, bool) {
 // by the OCM provider. (Returns a random machine type if the config suggests it to be random.)
 func (o *OCMProvider) DetermineMachineType(cloudProvider string) (string, error) {
 	computeMachineType := viper.GetString(ComputeMachineType)
+	returnedType := ""
 
 	// If a region is set to "random", it will poll OCM for all the regions available
 	// It then will pull a random entry from the list of regions and set the ID to that
-	if computeMachineType == "random" {
+	if computeMachineType == "random" && rand.Intn(3) > 2 {
```

Comment:
This can never be true. `Intn` returns a number less than the provided integer, so the highest possible value is 2.
```suggestion
	if computeMachineType == "random" && rand.Intn(3) >= 2 {
```